bitnami/rabbitmq/README.md

```diff
@@ -83,6 +83,7 @@ The following table lists the configurable parameters of the RabbitMQ chart and
 | `auth.tls.serverCertificate`              | Server certificate content                                                                                           | `nil`                                                        |
 | `auth.tls.serverKey`                      | Server private key content                                                                                           | `nil`                                                        |
 | `auth.tls.existingSecret`                 | Existing secret with certificate content to RabbitMQ credentials                                                     | `nil`                                                        |
+|`auth.tls.existingSecretFullChain`         | Whether or not the existing secret contains the full chain in the certificate (`tls.crt`). Will be used in place of `ca.cert` if `true`. | `false`|
```

Comment:
Should we document how to use this as a subsection of the `### Enabling TLS support` section? I mean, sth like:
```
#### TLS integration with cert-manager

...
```
And provide some basic instructions to use cert-manager to generate the certificates, and how to use them on the RabbitMQ chart

Reply:
Good call, added. Since I have only confirmed this issue with Let's Encrypt certificates, I made sure to specify such in the docs.